src/.nuget/packageLoad.targets

```diff
@@ -129,15 +130,18 @@
           Inputs="@(ProjectJsonFile)"
           Outputs="%(ProjectJsonFile.RootDir)%(ProjectJsonFile.Directory)project.lock.json"
           AfterTargets="_RestoreGlobalReferences"
-          Condition="('$(SolutionFile)'=='' or !Exists('$(SolutionFile)')) and '$(OS)'=='Unix'"  >
+          Condition="('$(SolutionFile)'=='' or !Exists('$(SolutionFile)')) and '$(OS)'!='Windows_NT'"  >
     <PropertyGroup>
       <ProjectDirectory>%(ProjectJsonFile.RootDir)%(ProjectJsonFile.Directory)</ProjectDirectory>
     </PropertyGroup>
 
     <Message Importance="High" Text="Restoring in $(ProjectDirectory)" />
 
-    <!-- Restore packages for this directory -->
-    <Exec Command="$(NugetRestoreCommand) $(NuGetPackageRestoreOptions) -PackagesDirectory &quot;$(HOME)/.nuget/packages&quot; &quot;@(ProjectJsonFile)&quot;" />
+    <!-- Restore packages for this directory using Nuget-->
+    <!-- <Exec Command="$(NugetRestoreCommand) $(NuGetPackageRestoreOptions) -PackagesDirectory &quot;$(HOME)/.nuget/packages&quot; &quot;@(ProjectJsonFile)&quot;" />  -->
```

Comment:
This target is only run on non Windows.
Nuget restore does not quite work momentarily, so I am running restore with DNU.
I left the Nuget line commented out because we'll have to re-enable when net core Nuget comes out.

Another option is to split this target in 2 targets (restoreDNU and restoNuget) and then have a master target select between the two. But I thought that might be overkill.